### PR TITLE
Add taylor

### PR DESCRIPTION
**Pull request by tsmit291**
Add bio + headshot

### CODE REVIEW

**Comment by josh-padnick** [comment]
@eak12913 @mcalhoun How do we deploy this?

File: _data/team.yml [hunk 1]
```diff
@@ -144,6 +144,14 @@
   bio: |
     Marina is a Brazilian software developer, currently living in Berlin and and now working as a Software Engineer at Gruntwork. She has worked across a variety of industries, including healthcare, marketing, finance, and car trading, and across the entire stack, including backend, frontend, DevOps, and testing. She loves functional programming in Clojure, travel, and video games.
 
+- name: Taylor Smith
+  url: "https://www.linkedin.com/in/tsmit291/"
+  title: Product Manager
+  image: /assets/img/headshots/taylor-smith.jpg
+  bio: |
+    Taylor is a Product Manager with a passion for bridging the gap between customer needs and business. She previously worked for a Fortune 6 company where she focused on facilitating Product Management during a digital transformation. Extremely detail oriented and creative, in her free time you can find her tackling various interior design projects at home, or spending time with her fiancé and dog, Lewis.
```
Comment (josh-padnick) [inline _data/team.yml]: Sounds good! You may 
```suggestion
    Taylor is a Product Manager with a passion for bridging the gap between customer needs and business. She previously worked for a Fortune 6 company where she focused on facilitating Product Management during a digital transformation. Extremely detail oriented and creative, in her free time you can find her tackling various interior design projects at home, or spending time with her fiancé or her dog, Lewis.
```

Comment (josh-padnick) [reply _data/team.yml]: This read a little bit like it's Lewis you'll be marrying!

**Comment by eak12913** [comment]
Merging and deploying this now!

**Pull request merged**